packages/rich-text-structure/src/readme.md

```diff
@@ -0,0 +1,35 @@
+# Rich Text Structure
+
+This module contains helper functions to convert HTML or a DOM tree into a rich text structure and back, and to modify the structure with functions that are similar to `String` methods, and some additional ones for formatting.
+
+## `create( element, range, multilineTag, settings )`
+
+## `toHTMLString( record, multilineTag )`
+
+## `apply( record, current, multilineTag )`
+
+## `isCollapsed( record )`
+
+## `isEmpty( record )`
+
+## `applyFormat( record, format, startIndex, endIndex )`
+
+## `removeFormat( record, formatType, startIndex, endIndex )`
+
+## `getActiveFormat( record, formatType )`
+
+## `getTextContent( record )`
+
+## `slice( record, startIndex, endIndex )`
+
+## `replace( record, pattern, replacement )`
+
+## `insert( record, recordToInsert, startIndex, endIndex )`
+
+## `remove( record, startIndex, endIndex )`
+
+## `split( record, startIndex, endIndex )`
+
+## `join( records, separator )`
+
+## `concat( ...records )`
```

Comment:
an explanation with examples would be insanely valuable here. I don't think it's obvious what all these are for, what `start` and `end` mean, etc…

Reply:
Yeah. This file was just a stub to be expanded.